apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/ValueConversion.cs

```diff
@@ -373,14 +441,15 @@ private static object ConvertToClrTypeImpl(Value wireValue, V1.Type spannerType,
                 }
                 //a bit of recursion here...
                 IDictionary dictionary = (IDictionary)Activator.CreateInstance(targetClrType);
+                var itemType = targetClrType.GetGenericArguments().Skip(1).FirstOrDefault() ?? typeof(object);
```

Comment:
We could run into interesting problems here if what's passed in is an open generic type, but I think it's probably reasonable for that to fail.